.travis.yml

```diff
@@ -1,20 +1,15 @@
-# This config file for Travis CI utilizes ros-industrial/industrial_ci package.
-# For more info for the package, see https://github.com/ros-industrial/industrial_ci/blob/master/README.rst
-sudo: required 
-dist: trusty 
-services:
-  - docker
-language: generic 
-compiler:
-  - gcc
+# Travis CI instructions
+sudo: required
+dist: trusty
+language: generic
+
 env:
   matrix:
-    - ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu NOT_TEST_BUILD=true NOT_TEST_INSTALL=true
-    - ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu NOT_TEST_BUILD=true NOT_TEST_INSTALL=true
-    - ROS_DISTRO="kinetic"  ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu NOT_TEST_INSTALL=true
-    - ROS_DISTRO="kinetic"  ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu NOT_TEST_INSTALL=true
```

Comment:
I can't be sure yet, but I believe there was a reason for the `NOT_TEST_INSTALL` and other env vars there.

Reply:
I tested this configuration locally (docker container) and worked.. I can re-add the `NOT_TEST_INSTALL=true` but I think is not needed anymore

Reply:
Ok. Let's remove it then and keep an eye on it.

We can always re-add it later.